root/components/ReleaseLabelList.js

```diff
@@ -21,6 +21,8 @@ type ReleaseLabelsProps = {
   +labels?: $ReadOnlyArray<ReleaseLabelT>,
 };
 
+const getLabelGid = x => x.gid;
```

Comment:
Why create a new const for one use?

Reply:
This way it can use the same (hopefully optimized, since it runs on the same type of object) function in memory every time the component is called, rather than creating a new function each time. (Probably a drop in the bucket and I'd do it the other way if it helped readability, but the named function actually seems more readable to me, too.)